components/Navbar.tsx

```diff
@@ -21,8 +21,8 @@ import MenuIcon from "@mui/icons-material/Menu";
 import CategoryDropDownMenu from "./CategoryDropDownMenu";
 
 const drawerWidth = 240;
-const navItems = ["Home", "Sign In", "Donation"];
-const links = ["/", "/auth/sign-in", "/donation-info"];
+const navItems = ["Home", "Sign In", "Donation","Contact"];
+const links = ["/", "/auth/sign-in", "/donation-info","/contact"];
```

Comment:
There is an inconsistency between the element in the links array and the contact page folder, which is causing the contact-page to be unresponsive.